api/src/test/java/org/openmrs/notification/MessageServiceTest.java

```diff
@@ -113,4 +118,107 @@ public void sendMessage_shouldSendMessage() throws MessageException {
 		}
 	}
 	
+	/**
+	 * @see MessageService#getMessagePreparator()
+	 */
+	@Test
+	public void getMessagePreparator_shouldGetMessagePreparator() {
+		
+		assertNotNull(ms.getMessagePreparator());
+		
+	}
+	
+	/**
+	 * @see MessageService#setMessagePreparator(MessagePreparator)
+	 */
+	@Test
+	public void setMessagePreparator_shouldSetMessagePreparator(){
+		
+		ms.setMessagePreparator(null);
+		assertNull(ms.getMessagePreparator());
+		
+	}
+	
+	/**
+	 * @throws MessageException
+	 * @see MessageService#prepareMessage(Template)
+	 */
+	@Test
+	public void prepareMessage_shouldPrepareMessage() throws MessageException{
```

Comment:
lets work together on the other PR before we get into this one

Reply:
@teleivo The coveralls showed up in red for these methods, and that was why I wrote  test methods. Thank you very much for your reviews.